osv/test_tools/test_repository.py

```diff
@@ -0,0 +1,158 @@
+"""test_repository"""
+import pygit2
+import json
+from datetime import datetime
+from enum import Enum
+import os
+import shutil
+import uuid
+
+
+class TestRepository:
+  """ Utilitary class to create a test repository for the git tests
+  """
+
+  class VulnerabilityType(Enum):
```

Comment:
For future-proofing and maintenance, could you try and use `osv.vulnerability_pb2` and use `vulnerability_pb2.Event.DESCRIPTOR.fields_by_name` to introspect into the OSV record definition code, rather than having this standalone definition that could go out of sync with what's defined in https://github.com/google/osv.dev/blob/ad0eed1cdb155bedbbe195f049b32daf459deca5/osv/vulnerability.proto#L67-L77

See also https://googleapis.dev/python/protobuf/latest/google/protobuf/descriptor.html